packages/app/src/vis-packs/core/line/config.tsx

```diff
@@ -50,13 +53,7 @@ function createStore() {
       }),
       {
         name: 'h5web:line',
-        whitelist: [
-          'curveType',
-          'showGrid',
-          'xScaleType',
-          'yScaleType',
-          'showErrors',
-        ],
+        partialize: (state) => omit(state, ['autoScale']),
```

Comment:
`whitelist` is deprecated in favour of a generic `partialize`, which can be used to filter out the properties we're not interested in persisting.